modules/gitops-release-notes-1-8-2.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-2_{context}"]
+= Release notes for {gitops-title} 1.8.2
+
+{gitops-title} 1.8.2 is now available on {product-title} 4.10, 4.11, and 4.12.
+
+[id="fixed-issues-1-8-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, when you configured Dex using the `.spec.dex` parameter and tried to log in to the Argo CD UI by using *Log In Via Openshift*, you were not able to log in. This update fixes the issue. 
```

Comment:
It's normally "OpenShift", but this is a UI reference. Could you double-check if that particular UI has "Openshift" or "OpenShift"? If it is not possible to know for sure, I would use "OpenShift".

Reply:
@mramendi yes, I have checked and rectified it in the revised PR. Thank you!